README.md

```diff
@@ -22,6 +22,24 @@ Taking advantage of [OWASP ZAP](https://www.zaproxy.org/) and additional scanner
 
 Linux and MacOS are supported.
 
+#### Note regarding MacOS and ZAP
+
+* in "none" mode: you will need to configure `scanners.zap.container.parameters.executable` to the installation path of the `zap.sh` command, because it is not available in the PATH. Usually, its path is `/Applications/OWASP ZAP.app/Contents/Java/zap.sh`
```

Comment:
Since this part comes prior to the section that explains "none" mode, chances are a reader won't have idea what it means. it'd be nice to mention 'general.config.type' clearly and  add the link to https://github.com/RedHatProductSecurity/rapidast#choosing-the-execution-environment for reference.

Reply:
clarified